src/languages/golfscript/index.ts

```diff
@@ -63,10 +68,15 @@ const golfscriptLanguage: Language = {
   emitter: emitProgram,
   phases: [
     search(hardcode()),
-    required(printIntToPrint),
+    required(
+      printIntToPrint,
+      implicitlyConvertPrintArg,
+      printLnToPrint,
+      printConcatToMultiPrint,
```

Comment:
Do you like this more than a single plugin?

Reply:
I believe so, since `printConcatToMultiPrint` can also golf concats that weren't generated by `printLnToPrint`, like `print (concat $a $b);`